aes.js

```diff
@@ -81,14 +81,17 @@ AES.keySize = 256 / 8
 AES.prototype.keySize = AES.keySize
 
 function bufferToArray (buf) {
-  var len = buf.length / 4
+  if (!Buffer.isBuffer(buf)) {
+    throw new TypeError('Input must be a Buffer')
+  }
+  var len = (buf.length / 4) | 0
```

Comment:
can we have a comment here about the exact behavior of this? it converts to int32 via truncation right ?